nixarr/lib/api-keys.nix

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This should probably live in util, right?

Reply:
There's now also the `nixarr list-api-keys` command, you should check it out for inspiration